.github/workflows/nightly.yml

```diff
@@ -513,7 +513,7 @@ jobs:
         printf 'FROM scratch\nENV EDGEDB_USER edgedb\nADD edgedb /usr/bin/\nENTRYPOINT ["/usr/bin/edgedb"]\n' \
           >dockerctx/Dockerfile
         tag=$(date +%Y%m%d%H%M%S)$(echo "${GITHUB_SHA}" | cut -c1-6)
-        echo "SNAPSHOT_TAG=${tag}" >> $GITHUB_ENV
+        echo ::set-env name=SNAPSHOT_TAG::${tag}
```

Comment:
The other version is correct. ::set-env is deprecated for security reasons

Reply:
Interesting. I only compiled what's in the templates so I'll be changing it then.